cfpb/cfgov-refresh
# Remove chosen

Removes the Chosen.js plugin from the code base in favor of our new multi-select, includes fixes for improper option values from @kurtw 

## Additions

- WordPress processors for proper option values

## Removals

- Removes Chosen library and it's initialization and styles

## Changes

- Updates common router to initialize the multi-select script on any select with the `multi` attribute
- Fixes the values for author and tag options to remove special characters

## Testing

- checkout branch and run `python cfgov/manage.py sheer_index`
- run `gulp build`
- navigate to `/blog`, `/newsroom`, and `/activity-log` and check out the two multi-select options

## Review

- @schaferjh 
- @duelj 
- @sebworks 
- @KimberlyMunoz 
- @anselmbradford 

## Screenshots

<img width="862" alt="screen shot 2016-03-11 at 12 58 21 pm" src="https://cloud.githubusercontent.com/assets/1280430/13710738/f9d70356-e788-11e5-9c7c-6b352c9a801a.png">

## Todos

- There's a bug in the multi-select where selected options aren't displayed after form submission (though the filters are applied). Bugfix for that in a forthcoming PR.

## Checklist
